Client/Application/AppDelegate.swift

```diff
@@ -135,6 +142,52 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         SDWebImageDownloader.sharedDownloader().setValue(firefoxUA, forHTTPHeaderField: "User-Agent")
     }
+
+    func application(application: UIApplication, handleActionWithIdentifier identifier: String?, forLocalNotification notification: UILocalNotification, completionHandler: () -> Void) {
+        if let actionId = identifier {
+            if let action = SentTabAction(rawValue: actionId) {
+                switch(action) {
+                case .View:
+                    viewURLInNewTab(notification)
+                    break
+                case .Bookmark:
+                    addBookmark(notification)
+                    break
+                case .ReadingList:
```

Comment:
There were some comments in the code that said 
`// TODO: set additionalActions to bookmark or add to reading list.`
So I just did that seeing as I was there. Happy to remove it though.